src/iqsim.jl

```diff
@@ -8,7 +8,7 @@
           overlap::NTuple{N,<:Real}=ntuple(i->1/6,N),
           soft::AbstractVector=[], hard::Dict=Dict(), tol::Real=.1,
           path::Symbol=:raster, nreal::Integer=1,
-          threads::Integer=cpucores(), gpu::Bool=false,
+          threads::Integer=cpucores(),
```

Comment:
You missed the resource option here?

Reply:
Yes. I forgot to include it in the documentation.

Reply:
You added the option in the wrong place in the docstring. As we discussed above, we strive for consistency and so this option should appear before the options `threads`.